rails/spec/models/external_activity_spec.rb

```diff
@@ -420,11 +424,11 @@
   end
 
   # TODO: auto-generated
-  describe '#duplicate_on_lara' do
-    xit 'duplicate_on_lara' do
+  describe '#duplicate_on_remote' do
+    xit 'duplicate_on_remote' do
       external_activity = described_class.new
       root_url = 'root_url'
-      result = external_activity.duplicate_on_lara(root_url)
+      result = external_activity.duplicate_on_remote(root_url)
```

Comment:
I think its best to just remove this '#duplicate_on_remote' block.  The method is tested by the duplicate specs above. And this spec it disabled with the `xit`.